generate/generate.go

```diff
@@ -417,8 +417,12 @@ func (as *allServices) GeneralCode() ([]byte, error) {
 	pn("  if err := json.Unmarshal(b, &m); err != nil {")
 	pn("    return nil, err")
 	pn("  }")
-	pn("  for _, v := range m {")
-	pn("    return v, nil")
+	pn("  for k, v := range m {")
+	pn("    if k == \"count\" {")
+	pn("		continue")
+	pn("	} else {")
+	pn("		return v, nil")
+	pn("	}")
```

Comment:
Don't know about this one... Wouldn't you loose the `count` info returned from a Listxxx API call this way?

Could you share some info why you added this part?